esphome/components/remote_transmitter/__init__.py

```diff
@@ -74,13 +74,11 @@ async def to_code(config):
                 cg.add(var.set_clock_resolution(config[CONF_CLOCK_RESOLUTION]))
             if CONF_USE_DMA in config:
                 cg.add(var.set_with_dma(config[CONF_USE_DMA]))
-            if CONF_ONE_WIRE in config:
-                cg.add(var.set_one_wire(config[CONF_ONE_WIRE]))
             if CONF_EOT_LEVEL in config:
                 cg.add(var.set_eot_level(config[CONF_EOT_LEVEL]))
-            elif CONF_ONE_WIRE in config and config[CONF_ONE_WIRE]:
+            elif config[CONF_PIN][CONF_MODE][CONF_OPEN_DRAIN]:
                 cg.add(var.set_eot_level(True))
-            elif CONF_INVERTED in config[CONF_PIN] and config[CONF_PIN][CONF_INVERTED]:
+            elif config[CONF_PIN][CONF_INVERTED]:
```

Comment:
Could this and the line above be combined to use an `or`?:
```suggestion
            elif config[CONF_PIN][CONF_MODE][CONF_OPEN_DRAIN] or config[CONF_PIN][CONF_INVERTED]:
```

Reply:
I could but the formatter tool would change that into an ugly 4 line if, not sure that is better:
```
+            elif (
+                config[CONF_PIN][CONF_MODE][CONF_OPEN_DRAIN]
+                or config[CONF_PIN][CONF_INVERTED]
+            ):
```

Reply:
I think that's fine, I'm used to seeing it do that.

Reply:
Ok think this looks the least ugly :)